tex/complex-ana/quintic.tex

```diff
@@ -0,0 +1,193 @@
+\chapter{Bonus: Topological Abel-Ruffini Theorem}
+We've already shown the Fundamental Theorem of Algebra.
+Now, with our earlier intuition on holomorphic $n$th roots,
+we can now show that there is no general formula for
+the roots of a quintic polynomial.
+
+\section{The Game Plan}
+Firstly, what do we even mean by ``formula'' here?
+
+\begin{definition}
+	A \vocab{quintic formula} would be a formula taking in the coefficients
+	$(a_0, \dots, a_5)$ of a degree $5$ polynomial $P$,
+	using the operations $+$, $-$, $\times$, $\div$, $\sqrt[n]{}$
+	finitely many times, that maps to the five roots $(z_1, \cdots, z_5)$ of $P$.
+\end{definition}
+
+Now, any proposed quintic formula $F$ receives the same coefficients when the roots are the same,
+and thus gives the same output. This is fine at first glance, but swapping two roots continuously
+might pose more issues. $F$ must create and preserve the order of the roots under these permutations.
+
+\begin{ques}
+	Convince yourself this is true.
```

Comment:
What does "this" refer to here?

Reply:
That $F$ basically "tracks" the order of the roots as you swap them along a path.

Reply:
Gotcha, can we edit the question to make it more clear what "this" is pointing to

Reply:
I've tried to do so.